libnavigation-ui/src/test/java/com/mapbox/navigation/ui/NavigationViewModelTest.java

```diff
@@ -85,48 +71,16 @@ public void updateRoute_navigationIsNotUpdatedWhenChangingConfigurations() {
     DirectionsRoute route = mock(DirectionsRoute.class);
     MapConnectivityController mockedConnectivityController = mock(MapConnectivityController.class);
     MapOfflineManager mapOfflineManager = mock(MapOfflineManager.class);
-    NavigationViewRouter router = mock(NavigationViewRouter.class);
+    NavigationViewOptions mockNavigationViewOptions = mock(NavigationViewOptions.class);
     NavigationViewModel viewModel = new NavigationViewModel(application, navigation, mockedConnectivityController,
-            mapOfflineManager, router);
+            mapOfflineManager, mockNavigationViewOptions);
     viewModel.onDestroy(true);
 
-    viewModel.updateRoute(route);
+    //viewModel.retrieveRoute().
 
-    verify(navigation, times(0)).startNavigation(route);
-  }
-
-  @Test
-  public void navigationRouter_onDestroyInvokedWhenViewModelIsDestroyed() {
-    Application application = mock(Application.class);
-    MapboxNavigation navigation = mock(MapboxNavigation.class);
-    MapConnectivityController mockedConnectivityController = mock(MapConnectivityController.class);
-    MapOfflineManager mapOfflineManager = mock(MapOfflineManager.class);
-    NavigationViewRouter mockedRouter = mock(NavigationViewRouter.class);
-    NavigationViewModel viewModel = new NavigationViewModel(application, navigation, mockedConnectivityController,
-            mapOfflineManager, mockedRouter);
-    viewModel.onCleared();
-    verify(mockedRouter).onDestroy();
+    verify(navigation, times(0)).startTripSession();
   }
 
-  @Test
-  public void updateRoute_navigationIsUpdated() {
```

Comment:
Is it safe to remove this test? I mean, is this scenario still covered by other test or was the functionality removed?